packages/wrangler/src/d1/execute.tsx

```diff
@@ -249,12 +265,8 @@ async function executeRemotely(
 	);
 
 	if (shouldPrompt) {
+		//only log if we should prompt
 		logger.log(`🌀 Executing on ${name} (${db.uuid}):`);
-
-		// Don't output if shouldPrompt is undefined
-	} else if (shouldPrompt !== undefined) {
```

Comment:
@geelen any strong feelings about me removing logger.error here?

Reply:
Yep that was just me hacking it so that `| jq` kinda worked, rip all that out as needed.